src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/CheckPowTransactionRuleTest.cs

```diff
@@ -49,7 +49,7 @@ public void CheckTransaction_TransactionOverSized_ThrowsBadTransactionOversizeCo
         {
             var transaction = new Transaction();
             transaction.Inputs.Add(new TxIn());
-            transaction = GenerateTransactionWithWeight(transaction, this.options.MaxBlockBaseSize + 1, NetworkOptions.TemporaryOptions & ~NetworkOptions.Witness);
+            transaction = GenerateTransactionWithWeight(transaction, this.options.MaxBlockBaseSize + 1, TransactionOptions.All & ~TransactionOptions.Witness);
```

Comment:
Is this not None? `TransactionOptions.All & ~TransactionOptions.Witness`

Reply:
In this case it is but if other flags would be added then it would not necessarily be.